### ISSUE TITLE
Bug: $hasPermsInChannel doesn't work...

### ISSUE THREAD

**Issue by ItzMann** [aoijs/aoi.js#442]
### Bug Type

Function

### Bugged Function

```js
module.exports = [{
     name: "hasperms",
     code: `
$hasPermsInChannel[$channelID;$authorID;sendmessages]
`
}]
```

### Version of aoi.js

v6-dev

### Version of Node.js

v18.14.0 (LTS)

### Excepted Output

The code should've returned `true` or `false`

### Relevant log output

```shell
But it return:
`upms.includes is not a function`
in console
```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by USERSATOSHI** [comment]
@Leref is this in your commits?

**Comment by Leref** [comment]
Resolved in 84bb8cfc7e7411998953d07df390b7b227b28db6.
